libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerNosara.java

```diff
@@ -782,6 +790,12 @@ public static String getEventNameForStat(AnalyticsTracker.Stat stat) {
                 return "plugin_directory_opened";
             case OPENED_PLUGIN_LIST:
                 return "plugin_list_opened";
+            case OPENED_PLANS:
+                return "site_menu_opened";
+            case OPENED_SHARING_MANAGEMENT:
+                return "site_menu_opened";
```

Comment:
Is `OPENED_SHARING_MANAGEMENT` necessary?  I see it's been added in [`AnalyticsTracker`](https://github.com/wordpress-mobile/WordPress-Android/pull/8616/files#diff-578f7e9e6f43c37a5ca1ee823a493989R227), but I don't see it being triggered anywhere.  Also, should the `"site_menu_opened"` value be the same as `OPENED_PLANS` above it?  If so, should we consolidate those into a `OPENED_SITE_MENU` case?

Reply:
Yeah. `OPENED_SHARING_MANAGEMENT` is the stat missing from that empty line.  Mea culpa.

Re: consolidation - I don't think so (if I've understood the question). Each of the `site_menu_opened` events has a predefined `menu_item` event property that indicates the specific menu item the event represents.  As far as tracks is concerned we're reusing a single event but we still need the value of the `menu_item` event property to be distinct.  Does that make sense? (Or did I misunderstand the question 😬 )